import{a5 as l,ai as c,ah as t,a8 as e,ae as i,a7 as s,aw as o,ax as a,aa as n}from"./index.8ae906f5.js";import{_ as r}from"./plugin-vue_export-helper.21dcd24c.js";var d="/assets/2023-05-2211.44.36.f159fab8.png",u="/assets/stitching.f8575d20.jpeg",_="/assets/2023-05-2200.08.08.6679af58.png",h="/assets/2022-11-0209.20.22.627e6d5e.png";const g={},f={class:"main",style:{"text-align":"center"}},p=t("img",{src:d,style:{width:"100%"}},null,-1),m={class:"row justify-center"},v=t("br",null,null,-1),x={style:{"text-align":"left"}},y={class:"row q-gutter-md justify-center"},C=t("img",{src:u},null,-1),b=t("div",{class:"text-h6"},"Purpose",-1),q=t("img",{src:_},null,-1),S=t("img",{src:h},null,-1),w=t("div",{class:"text-h6"},"Result",-1),z=t("div",{class:"text-h6"},"Requirements",-1),R=t("p",null," 1.Sample information: sample name, target gene, group name, gRNA sequence, barcodes. ",-1),j=t("p",null,"2.Sequencing data in Fastq.",-1),B=t("p",null,"3.Target gene sequence in Fasta.",-1),P=t("div",{class:"text-h6"},"Contact",-1),I=t("div",{class:"text-h6"},"Citation",-1),N=t("a",{href:"https://doi.org/10.1016/j.xplc.2023.100783"},"CrisprStitch: Fast evaluation of the efficiency of CRISPR editing systems.",-1);function Q(Y,F){return l(),c("div",f,[p,t("div",m,[e(i,{color:"primary",to:"/input",label:"start",size:"xl",icon:"directions",flat:""})]),v,t("div",x,[t("div",y,[e(o,{class:"my-card col-lg-4 col-auto col-md-8 col-xs-10"},{default:s(()=>[C,e(a,null,{default:s(()=>[b]),_:1}),e(a,{class:"q-pt-none"},{default:s(()=>[n(" CrisprStitch processes the high-throughput genome-editing data of CRISPR nuclease transformed protoplasts with integrated analysis steps, including mapping reads and normalizing reads count, calculating mutation frequency (deletion and insertion), evaluating efficiency and accuracy of genome-editing, and visualizing the results (tables and histograms). ")]),_:1})]),_:1}),e(o,{class:"my-card col-lg-4 col-auto col-md-8 col-xs-10"},{default:s(()=>[q,S,e(a,null,{default:s(()=>[w]),_:1}),e(a,null,{default:s(()=>[n(" After data processing, the application provides an overview of different mutation results in tables and plots. Deletion, deletion size, insertion, insertion size, and substitution are included. ")]),_:1})]),_:1}),e(o,{class:"my-card col-lg-3 col-auto col-md-8 col-xs-10"},{default:s(()=>[e(a,null,{default:s(()=>[z]),_:1}),e(a,{class:"q-pt-none"},{default:s(()=>[R,j,B]),_:1}),e(a,null,{default:s(()=>[P]),_:1}),e(a,null,{default:s(()=>[e(i,{flat:"",color:"secondary",href:"mailto:hanyangshuo@zhangtaolab.org"},{default:s(()=>[n("hanyangshuo@zhangtaolab.org")]),_:1})]),_:1}),e(a,null,{default:s(()=>[I,n(" Han YS\u2020, Liu GQ\u2020, Wu YC, Bao Y, Zhang Y*, Zhang T*. "),N,n(" Plant Communications DOI: https://doi.org/10.1016/j.xplc.2023.100783 ")]),_:1})]),_:1})])])])}var k=r(g,[["render",Q]]);export{k as default};
